.changeset/flat-bottles-flash.md

```diff
@@ -0,0 +1,7 @@
+---
+"@khanacademy/wonder-blocks-switch": minor
```

Comment:
Would removing support for the `khanmigo` theme be a breaking change? If so, should this be a `major` version?

Reply:
I considered that, but in this case it is not a breaking change per se (yet) as the `wb-theming` package still supports the `khanmigo` theme underneath. This is one of the cases where this implementation doesn't break the component directly.